README.md

```diff
@@ -24,15 +24,78 @@ In this challenge, create a web API around the following resources: `Projects` a
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
+
+
 - [ ] Mention two parts of Express that you learned about this week.
+const express = require('express');
+const server = express();
+server.listen(3000, () =>
+  console.log('Example app listening on port 3000!'),
+);
+
+
+
 
 - [ ] Describe Middleware?
+Middleware functions are functions that have access to the request object (req), the response object (res), and the next middleware function in the application’s request-response cycle. The next middleware function is commonly used by a variable named next.
```

Comment:
To be more specific, in Express, middleware is code that runs in the middle of (or between) the request and the reponse code running on an Express server.